draft-pauly-httpbis-geoip-hint.md

```diff
@@ -27,33 +27,55 @@ author:
 
 --- abstract
 
-This documents defines an HTTP Client Hint that allows a client to share information
-about its IP Geolocation. This helps ensure that servers have information about location
-that is consistent with what a client expects and what other servers use.
+This document defines an HTTP Client Hint that allows the client to
```

Comment:
I think this abstract would be more powerful if we reversed the order to be (1) problem then (2) solution.

So something that paraphrases to "We're doing a lot more IP-hiding to improve privacy these days, but that costs a lot of IPs and also users often get bad geo results so users are sad. We fix this by sending the geo from client to origin. We accomplish that with a client hint"

Reply:
I rephrased the introduction to reflect this. LMK what you think!